gem/lib/phlexing/converter.rb

```diff
@@ -23,6 +23,8 @@ def initialize(source = nil, **options)
       convert(source)
     end
 
+    # private
```

Comment:
I added this to indicate that we should try to make the below methods private. I believe they are required in tests for now.

Reply:
I left them public on purpose, so you as a user of the gem can decide what part you want. But I guess it would be enough if we would only keep the `code` one public

Reply:
What's confusing here is that there are two ways to use it. You either pass an option and look at `code` or you call call `component_code` or `template_code` directly and it ignores the option.

Reply:
I would either remove the two methods or make them private

Reply:
👍 I’ll work on that in a new PR.